calico_containers/pycalico/netns.py

```diff
@@ -180,6 +179,28 @@ def remove_ip_from_ns_veth(container_pid, ip, veth_name_ns):
                        "dev", "%(device)s" % veth_name_ns])
 
 
+def provision_veth(self, endpoint, pid, ip, veth_name_ns):
```

Comment:
Can we rename `pid` to `ns_pid`? Elsewhere in pycalico we refer to `container_pid`, `pid`, and `cpid` and the ambiguity is starting to get a little crazy. 

Limit this change only to the new functions you've introduced - no need to go running around the rest of the code base making this change now.